web/src/components/PlanWorkoutForm/PlanWorkoutForm.js

```diff
@@ -131,6 +127,20 @@ const PlanWorkoutForm = (props) => {
         </SelectField>
         <FieldError name="activityId" className="rw-field-error" />
 
+        <Label
+          name="isKeyWorkout"
+          className="rw-label"
+          errorClassName="rw-label rw-label-error"
+        >
+          Key workout?
+        </Label>
+        <CheckboxField
+          name="isKeyWorkout"
+          className="rw-input"
+          errorClassName="rw-input rw-input-error"
+          defaultChecked={props.planWorkout?.isKeyWorkout}
```

Comment:
This was sneaky -- I thought it'd be `defaultValue`, but that didn't work. I tracked down the correct prop name in [the PR that originally introduced `CheckboxField`](https://github.com/redwoodjs/redwood/pull/719/files#diff-ad72b10dba26b0f02b51f33ff7d8ed95212279033a78f54fe5b3d0f8c037f881R115).